Please fix the compliance failure:
```
Error: See docs.zephyrproject.org/latest/contribute/#commit-guidelines for more details

336a241ea20db990ccbc0d7d8b40463ec39bedc5: author email (voja <milivojevicvoja@yahoo.com>) needs to match one of the signed-off-by entries.
336a241ea20db990ccbc0d7d8b40463ec39bedc5: author email (voja <milivojevicvoja@yahoo.com>) does not follow the syntax: First Last <email>.
Error: See docs.zephyrproject.org/latest/contribute/#commit-guidelines for more details

3: UC4 Line exceeds max length (84>72): "corrent value is retured when using sensor channel get function to read z axis value"
Error: Process completed with exit code 1.
```

--- a/drivers/sensor/bmm150/bmm150.c
+++ b/drivers/sensor/bmm150/bmm150.c
@@ -353,7 +353,7 @@ static int bmm150_channel_get(const struct device *dev,
 		bmm150_convert(val, drv_data->sample_y);
 		break;
 	case SENSOR_CHAN_MAGN_Z:
-		bmm150_convert(val, drv_data->sample_x);
+		bmm150_convert(val, drv_data->sample_z);
 		break;
 	case SENSOR_CHAN_MAGN_XYZ:
 		bmm150_convert(val, drv_data->sample_x);

